File: integration-test/test-projects/multiCompositeBuild/TestCompositeApp/app/src/main/java/com/test/compositeapp/MainActivity.java
package com.test.compositeapp;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        com.test.composite1.mylibrary.Test.doSomething();
        com.test.composite2.mylibrary.Test.doSomething();
        com.test.composite3.mylibrary.Test.doSomething();
        com.test.composite4.mylibrary.Test.doSomething();
    }
}

File: integration-test/test-projects/transformInModuleWithKotlin/app/src/main/kotlin/com/android/tests/libstest/app/MainActivity.kt
package com.android.tests.libstest.app

import StringUtil.android
import com.android.tests.libstest.lib.Lib

import android.app.Activity
import android.os.Bundle
import java.util.logging.Logger

class MainActivity : Activity() {
    /** Called when the activity is first created.  */
    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)

        App.handleTextView(this)
        Logger.getLogger(Logger.GLOBAL_LOGGER_NAME).warning(
                Lib.someString().android())
    }
}
